Contest_Control_System_Requirements.md

```diff
@@ -1020,6 +1007,12 @@ The CCS must provice access to the Contest API including the
             "group_ids"
          ]
       },
+      { 
+         "type": "accounts", 
+         "properties": [
+            "id", "username", "type", "ip", "team_id"
+         ]
+      },
```

Comment:
I don't think we need to require the CCS to be able to export accounts.

Even if we do, I don't think we need to include "ip".

Reply:
I’m fine with dropping this. I would like to add IP to the import of you all agree

Reply:
What would that mean though? There is no requirement that requires the use of the information, so what does it mean that it must be imported?

Reply:
Are you explicitly referring to IP? I would like to add that as a requirement somewhere, but not sure where. I'm also fine not doing that though.